src/components/LinesControls.vue

```diff
@@ -179,50 +177,50 @@ export default {
       if (this.newDistance !== 0) {
         this.distance = this.newDistance;
         this.edited = moveAndExtendLine(
-          this._zincObject, this.currentIndex, this.newDistance, true) || this.edited;
+          this.zincObject, this.currentIndex, this.newDistance, true) || this.edited;
       } else {
         this.newDistance = this.distance;
       }
     },
     onLengthSliding: function() {
       this.newDistance = Math.pow(10, this.lengthScale) * this.distance;
       this.edited = moveAndExtendLine(
-        this._zincObject, this.currentIndex, this.newDistance, true) || this.edited;
+        this.zincObject, this.currentIndex, this.newDistance, true) || this.edited;
     },
     onMoveSliding: function() {
       const diff = (this.adjust - this.pAdjust) * this.distance;
       this.edited =  moveAndExtendLine(
-        this._zincObject, this.currentIndex, diff, false) || this.edited;
+        this.zincObject, this.currentIndex, diff, false) || this.edited;
       this.pAdjust = this.adjust;
     },
     reset: function() {
       this.adjust = 0;
       this.pAdjust = 0;
       this.lengthScale = 0;
-      this.distance = getLineDistance(this._zincObject, this.currentIndex);
+      this.distance = getLineDistance(this.zincObject, this.currentIndex);
       this.newDistance = this.distance;
       if (this.edited) {
-        this.$emit("primitivesUpdated", this._zincObject);
+        this.$emit("primitivesUpdated", this.zincObject);
         this.edited = false;
       }
     },
     setObject: function (object) {
       this.currentIndex = -1;
       this.distance = 0;
       if (object.isLines2) {
-        this._zincObject = object;
-        this.width = this._zincObject.getMorph().material.linewidth;
+        this.zincObject = markRaw(object);
+        this.width = this.zincObject.getMorph().material.linewidth;
         if (object.isEditable) {
           this.currentIndex = 0;
           this.distance = getLineDistance(object, this.currentIndex);
         }
       } else {
-        this._zincObject = undefined;
+        this.zincObject = undefined;
         this.linewidth = 10;
```

Comment:
A question: Should the `linewidth` here be `width`? Or is this a different variable?

Reply:
It should be width. Thanks for catching that.